trustwallet/wallet-core
# Codegen: preserve comments

## Description

Preserve and take over class/method boilerplate comments from input `TW*.h` files to generated source files.

## How to test

Check generated Swift/Java sources, check that comments are shown for methods in Xcode/AndroidStudio.

## Types of changes





* New feature (non-breaking change which adds functionality)



## Checklist




If you're adding a new blockchain
